tests/infrastructure/workload_availability/remediation_fencing_mhc/test_ha_vm.py

```diff
@@ -108,6 +108,7 @@ def wait_node_restored(node):
     wait_for_node_status(node=node)
 
 
+@pytest.mark.special_infra
```

Comment:
why is it needed here?

Reply:
I am planning to run all fencing tests together so i made it part of special_infra.

Reply:
but it doesn't require any special cluster config, right?
beside that it already has a `destructive` marker (which is excluded from t2)

Reply:
maybe i should let it run on PSI. okay i'll remove it.